RELEASE.md

```diff
@@ -89,8 +88,13 @@ Also the images we build are based on some image specified in the `FROM` stateme
     ```bash
     git checkout main
     git reset --hard <upstream>/main
-    git tag -a x.y.z -m x.y.z HEAD
-    git push --follow-tags <upstream> main
+    tbump x.y.z
+    ```
+
+  - [ ] Set the next prerelease version (don't create a tag).
+
+    ```bash
+    tbump --no-tag x.y+1.z-0.dev
```

Comment:
I'd like to stay with `x.y.z+1-0.dev` to avoid publishing early dev releases that sort later than upcoming patch releases.

If we end up doing a backport release with a patch release, because we have also added things we don't want to release yet with a minor/major bump, we could at that time also increment the minor or major version in the baseVersion field.

What do you think?

```suggestion
    tbump --no-tag x.y.z+1-0.dev
```

Reply:
I think that's fine. The trade off is we need to remember to bump baseVersion when we introduce the first enhancement / new feature (rather than just 'breaking' for the major version). Tracking baseVersion means every time we review a PR, we need to ask the questions:

1. have we already bumped the major baseVersion? (If yes, stop here)
2. Is this a breaking change? (If yes, bump major baseVersion)
3. have we already bumped minor baseVersion? (If yes, stop here)
4. Is this a new feature or enhancement? (If yes, bump minor baseVersion)

Let's try it and find out how we feel!

Reply:
Yepp I agree, that would be the procedure. When we merge the first thing into the default branch that motivates something beyond a patch release, we increment the baseVersion to represent that unless its already done.

If we would fail to remember bumping the baseVersion when we merge a PR that motivates a minor or major version bump, it wouldn't be a big deal in my mind. Those dev-releases following that would not correctly reflect the situation, but they would correctly order themselves in the semver2 version sorting.